src/5e-SRD-Monsters.json

```diff
@@ -22910,7 +22910,7 @@
             "url": "/api/ability-scores/con"
           },
           "dc_value": 19,
-          "success_type": "nhalfone"
+          "success_type": "half"
```

Comment:
Actually, I think this is slightly more broken. We're  missing the attack option for Repulsion Breath  which does have a `success_type` of `none`.

Reply:
@ogregoire This was not ready to be merged.

Reply:
Oh damn it Github. That's not how you expand files.

Reply:
Right, the description for Repulsion Breath from [the SRD](https://media.wizards.com/2016/downloads/DND/SRD-OGL_V5.1.pdf) is:

```
Repulsion    Breath.    The    dragon    exhales    repulsion    energy    
in    a    30-­‐foot    cone.    Each    creature    in    that    area    must    
succeed    on    a    DC    23    Strength    saving    throw.    On    a    
failed    save,    the    creature    is    pushed    60    feet    away    from    
the    dragon.
```

I haven't seen any way to describe this effect of a failed save on the value for `dc` itself.. (A possible solution might be the property `notes`, which has been used in other structures?) But it is in the description available under `desc` for the object for the action `Breath Weapons`, so there's at least a representation of the rule.

In any case, I reckon there should be a separate issue (and PR) to resolve this?

Reply:
(I restored the branch in case you want to revert it and fix the problem before merging the branch.)

Reply:
No no. You're fine. I got confused.